ios/AirPushNotification/CTAppController+APNAdditions.m

```diff
@@ -20,8 +20,11 @@
 @implementation CTAppController (APNAdditions)
 
 + (void)load {
-    
+    if([UNUserNotificationCenter class] != nil) {
```

Comment:
Isn't UNUserNotificationCenter only iOS 10? Does this class need to work on lower?